source/operations/monitoring.rst

```diff
@@ -28,9 +28,11 @@ collected metrics.
   trigger alerts based on collected metrics. See
   :ref:`minio-metrics-and-alerts-alerting` for more information.
 
-MinIO publishes collected metrics data using Prometheus-compatible data
-structures. Any Prometheus-compatible scraping software can ingest and
-process MinIO metrics for analysis, visualization, and alerting.
+When configured, the :ref:`MinIO Console <minio-console-metrics>` shows some metrics in the :guilabel:`Monitoring > Metrics` page.
+You can download these metrics as either ``.png`` images or ``.csv`` files.
+
+MinIO publishes collected metrics data using Prometheus-compatible data structures. 
```

Comment:
Move this up to the top and I'll steal/modify this as part of my ongoing metrics/monitoring reworks.

Reply:
Done.